src/Components/ProductTile.js

```diff
@@ -1,6 +1,15 @@
+import React from 'react';
+import { ReactSession } from 'react-client-session';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faChevronDown, faChevronUp } from '@fortawesome/free-solid-svg-icons';
 import { useHistory } from 'react-router';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8496218)
```suggestion
import DialogContentText from "@mui/material/DialogContentText";
```